docs/workforce-management-framework/topics/con-Security-Strategy-for-Keycloak.adoc

```diff
@@ -1,27 +1,28 @@
 [id='Security-Strategy-Keycloak-{chapter}']
-= Security Strategy for {WFM-RC-NameShort} Keycloak
+= Security Strategy for {WFM-RC-NameShort} {WFM-RC-KeycloakName}
 
 == Session Management
-Keycloak uses the Express middleware `Express Sessions` module for the {WFM-RC-NameShort} cloud server.
+{WFM-RC-KeycloakName} uses the Express middleware `Express Sessions` module for the {WFM-RC-NameShort} cloud server.
 Visit link:https://github.com/expressjs/session[express-session] to find more information about the available express
 session options.
-One of the key functions of Keycloak is you can administer sessions on the
-link:https://keycloak.gitbooks.io/documentation/server_admin/topics/sessions/administering.html[Keycloak server]
+One of the key functions of {WFM-RC-KeycloakName} is you can administer sessions on the
```

Comment:
I'm not gramar expert but I struggle to get any sense of this.

Reply:
One of the key functions of {WFM-RC-KeycloakName} is that you can administer sessions on the